apps/src/music/views/soundsPanel2.module.scss

```diff
@@ -0,0 +1,193 @@
+@import "color.scss";
+
+.soundsPanel {
+  user-select: none;
+  width: 600px;
+  height: 340px;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+
+  &Top {
+    display: flex;
+    justify-content: space-between;
+    margin: 5px 0;
+    user-select: none;
+
+    .segmentedButtons {
+      span {
+        font-size: 12px !important;
+      }
+    }
+  }
+
+  &Body {
+    display: flex;
+    flex-direction: row;
+    overflow: hidden;
+    flex: 1;
+    gap: 5px;
+
+    .leftColumn {
+      background-color: $neutral_dark;
+      width: calc(40% - 00px);
+      overflow-y: scroll;
+    }
+
+    .rightColumn {
+      background-color: $neutral_dark;
+      width: calc(67% + 00px);
+      overflow-y: scroll;
+      display: grid;
+      grid-template-columns: calc(33.3333% - 0px) calc(33.3333% - 0px) calc(33.3333% - 0px);
```

Comment:
nit: could widths just be `calc(100% / 3)`?

Reply:
Much better; done!